lib/std/zig/Ast.zig

```diff
@@ -465,6 +464,9 @@ pub fn renderError(tree: Ast, parse_error: Error, stream: anytype) !void {
                 }),
             }
         },
+        .ambiguous_operator_precedence => {
+            return stream.writeAll("ambiguous operator precedence, must use parentheses to disambiguate");
```

Comment:
```suggestion
            return stream.writeAll("ambiguous operator precedence; use parentheses to disambiguate");
```
The comma here is a bit strange IMO.